### PR TITLE
create DB and connect mongo shell

### PR DESCRIPTION
**Pull request by damienaries**
### What this PR does (required):
- connect to mongoDB
- To start mongo shell, paste this code from server directory in terminal `mongosh "mongodb+srv://cluster0.mxbsd.mongodb.net/dogsitters" --username callbackCats      `    
- Password: j30nBRVSuHreiyF8

### Screenshots / Videos (required):
- Screen Shot 2021-08-09 at 8 33 08 PM

### Any information needed to test this feature (required):
- Login to mongoAtlas with google account
      email callbackCats2021@gmail.com
      password Hatchways.?2021
- That will open our cluster. Within this our DB is called "dogsitters"
- Navigate to browse collection to see current database

- Mongo docs
  [(https://docs.mongodb.com/drivers/node/current/usage-examples/)]

### Any issues with the current functionality (optional):
- No issues so far I'm able to send data in from the shell

### CODE REVIEW

**Comment by VanVSJobs** [comment]
mongodb+srv://callbackCats:j30nBRVSuHreiyF8@cluster0.mxbsd.mongodb.net/dogsitters
if you want to connect using gui such as mongodb

File: server/sample.env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 JWT_SECRET=
-MONGO_URI=
-TEAM_NAMES=Bonnie,Sina,Damien,ja
\ No newline at end of file
+MONGO_URI="mongodb+srv://callbackCats:j30nBRVSuHreiyF8@cluster0.mxbsd.mongodb.net/dogsitters?retryWrites=true&w=majority"
```
Comment (bonnieli) [inline server/sample.env:2]: I would refrain from pushing up live credentials to this remote repository, Mongo Atlas will likely ping you to protect you from being hacked. instead, share this internally via Slack with your teammates :)

Comment (damienaries) [reply server/sample.env:2]: Ok I can send this uri with them on slack. if this is the sample.env can we also add it to gitignore to keep it private?

Comment (rajivtitus) [reply server/sample.env:2]: You don't want to add the sample.env to gitignore as other devs can use it as a boilerplate to create their local .env files :)

Comment (damienaries) [reply server/sample.env]: Received. I removed the credentials from there and pushed it

**Review by rajivtitus — COMMENTED** [review]
Hi @damienaries , please remove credentials from the PR template as well. Thank you!

**Comment by damienaries** [comment]
Done. Deleting branch

**Comment by rajivtitus** [comment]
> mongodb+srv://callbackCats:[j30nBRVSuHreiyF8@cluster0.mxbsd.mongodb.net](mailto:j30nBRVSuHreiyF8@cluster0.mxbsd.mongodb.net)/dogsitters
> if you want to connect using gui such as mongo compass

Hi @VanVSJobs , can you please edit/delete this comment as we are exposing sensitive information. Thanks!

**Comment by VanVSJobs** [comment]
I deleted it

**Pull request merged**